Products/CMFEditions/StandardModifiers.py

```diff
@@ -335,7 +335,7 @@ def persistent_id(obj):
             try:
                 # return a non None value if it is one of the object
                 # managers subobjects or raise an KeyError exception
-                return refs[id(obj)]
+                return refs[id(aq_base(obj))]
```

Comment:
aq_base was used on line 332 when we built a mapping with the ids of the children, so we have to use it here too.